starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort();
+    this.length += 1;
+  }
+  get(pos) {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.item[this.length - 1];
+    }
+  }
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(this.items);
+    }
+  }
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.min(this.items);
+    }
+  }
+  average() {
+    this.items.sum / this.length;
+  }
+  sum() {
+    this.items.reduce(function(acc, curr) {
```

Comment:
you actually need to return the reduce. I know it seems super weird to have two returns 

```
  sum() {
    return this.items.reduce(function(acc, curr) {
      return acc + curr;
    }, 0);
  }
```